<?xml version="1.0" ?>
<launch>
  <node name="tableDetectorOnLWR" pkg="lasa_perception_module" type="tableDetectorOnLWR" output="screen" respawn="false">
    
    <param name="camera_name" value="/camera"/>
    <param name="world_frame_name" value="/world_frame"/> 
    <param name="depth_cutoff" value="1.7"/> 
    <param name="plane_threshold" value="0.01"/> 
    <param name="output_point_cloud" value="/table"/>
    <param name="dough_pose_srv_name" value="/dough_Pose"/>
    <param name="dough_frame" value="/dough_frame"/>
    <param name="dough_attractor" value="/dough_attractor"/>

    <!-- Params On Boxy -->
<!--    <param name="camera_name" value="/head_xtion/"/>
    <param name="world_frame_name" value="/base_link"/> 
    <param name="depth_cutoff" value="1.5"/> -->
    
  </node>
</launch>